.nvmrc

```diff
@@ -1 +1 @@
-v6.10.2
+6.10.2
```

Comment:
Right, my `process.version` had a version number without the `v` for some reason but it's fixed now, I'll add it back